### PR TITLE
Issue 1

### PR DESCRIPTION
**Pull request by jsstevenson**

### CODE REVIEW

File: disease/schemas.py [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+
+
+class Disease(BaseModel):
+    """Define disease record."""
+
+    label: str
+    concept_id: str
+    aliases: Optional[List[str]]
+    other_identifiers: Optional[List[str]]
+    xrefs: Optional[List[str]]
```
[the remark this answers is not in the document]
Comment (korikuzma) [reply disease/schemas.py:45]: Not all of the concepts have these properties, but we should consider adding?

**Review by korikuzma — APPROVED** [review]
Good job!

File: disease/etl/ncit.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                logger.warning(f"No label for concept {concept_id}")
+                continue
+            aliases = node.P90
+            if aliases and label in aliases:
+                aliases.remove(label)
+
+            xrefs = []
+            if node.P207:
+                xrefs.append(f"{NamespacePrefix.UMLS.value}:"
+                             f"{node.P207.first()}")
```
[the remark this answers is not in the document]
Comment (jsstevenson) [reply disease/etl/ncit.py:155]: did you see any examples besides nci-m? there are PDQ references displayed online but for some reason they aren't in the source data

File: disease/schemas.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jsstevenson) [reply disease/schemas.py:45]: Alex indicated yesterday that we didn't need to worry about additional attributes for now with the exception of capturing some sort of "pediatric" flag (should be achievable via class inheritance structure) which I'm gonna break into a separate PR

File: disease/etl/ncit.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (korikuzma) [reply disease/etl/ncit.py:155]: I was looking at ICD-O-3 codes (P375) and IMDRF codes (oboInOwl:hasDbXref). Not sure if we need to include but thought I'd ask your opinion.

[the remark this answers is not in the document]
Comment (jsstevenson) [reply disease/etl/ncit.py:155]: Oh great catch!

**Pull request merged**